### PR TITLE
added clarifying note to credentialSchema section

### PR DESCRIPTION
**Pull request by brentzundel**
This PR is intended to address issue #474 
Signed-off-by: Brent <brent.zundel@gmail.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2092,6 +2092,13 @@ <h3>Data Schemas</h3>
           </dd>
         </dl>
 
+        <p class="note">
+The <code>credentialSchema</code> provides an opportunity to annotate type
+definitions or lock them to specific versions of the vocabulary. Authors of VCs
+can include a static version of their vocabulary via credentialSchema that is
+locked to some content integrity protection mechanism.
```
Comment (msporny) [inline index.html]: You may also want to make a point about doing syntactic checking on the credential as well using stuff like JSON Schema. Just reminding folks that this property has at least two uses.

Comment (msporny) [reply index.html]: In any case, approve of change whether or not you take the above into account.

Comment (brentzundel) [reply index.html]: Made some changes  in [cd62b44](https://github.com/w3c/vc-data-model/pull/533/commits/cd62b44daf6b6767d8eee86280c948096668de61)

**Comment by msporny** [comment]
I forget where we got to here. I think we agreed that we can add lots of non-normative text to this section, and @David-Chadwick said he'd do it... but are we:

1. Merging this PR and waiting for an additional one from @David-Chadwick, or
2. Closing this PR and waiting for @David-Chadwick to write a PR?

**Comment by David-Chadwick** [comment]
@msporny As I recall, you said you would write this text since you are more intimately knowledgeable about JSON-LD and its nuances. I said I would write the refresh text (which I did)

**Pull request merged**